examples/tictactoe/tsconfig.app.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./src/app.tsx","./src/main.tsx","./src/vite-env.d.ts","./src/components/board.tsx","./src/components/connectwallet.tsx","./src/components/gamelists.tsx","./src/components/newgame.tsx","./src/components/square.tsx","./src/constants/chains.ts","./src/constants/tictactoe.ts","./src/hooks/useacceptgame.ts","./src/hooks/usegames.ts","./src/hooks/usemakemove.ts","./src/hooks/usenewgame.ts","./src/hooks/useplayergames.ts","./src/types/game.ts"],"errors":true,"version":"5.6.2"}
```

Comment:
you can gitignore this and the node tsbuildinfo files https://github.com/ethereum-optimism/ecosystem/blob/acfd4b41e800214c5e56b3a408970c14c32eb332/.gitignore#L13C1-L13C24